client/src/Features/MicrosoftLearn/MicrosoftLearnSearch.tsx

```diff
@@ -15,16 +15,18 @@ export type MicrosoftLearnSearchStyles = SimpleComponentStyles<'root' | 'label'
 
 const MicrosoftLearnSearchInner = ({ styles }: IStylesOnly<MicrosoftLearnSearchStyles>): JSX.Element => {
   const learnStore = useStore('microsoftLearnStore');
+  const { filterStore, isLoadingCatalog } = learnStore;
 
   const classes = themedClassNames(styles);
   return useObserver(() => {
     return (
       <div className={classes.root}>
         <Label className={classes.label}>Search</Label>
         <SearchBox
-          onChange={(_e, newValue) => learnStore.updateSearchTerm(newValue || '')}

```

Comment:
I think you mentioned about adding them in the other PR review. I can remove them is they make it seem too crowded.

Reply:
Right, I asked them since we were adding `any` as the type to these entities, which was essentially wrong. Leaving them as is leaves a room for the type system to perform an auto-detection and pick up an appropriate type.